src/device.cpp

```diff
@@ -679,6 +679,7 @@ void cvk_device::build_extension_ils_list() {
         m_extension_string += ext.name;
         m_extension_string += " ";
     }
+    m_extension_string += config.device_extensions();
```

Comment:
As discussed, this approach will not work with applications that use `CL_DEVICE_EXTENSIONS_WITH_VERSION`. I suggest you add the extensions to the `m_extension` vector so both mechanisms are supported. I think it's acceptable to start by just hardcoding the version to 0.0.0.